core/vtk/ttkGeometrySmoother/ttkGeometrySmoother.cpp

```diff
@@ -11,7 +11,9 @@ vtkStandardNewMacro(ttkGeometrySmoother);
 
 ttkGeometrySmoother::ttkGeometrySmoother() {
   this->setDebugMsgPrefix("GeometrySmoother");
-
+#ifdef TTK_ENABLE_MPI
+  hasMPISupport_ = true;
```

Comment:
I'd recommend to move that to the constructor of the *base* class when available.
in this example `core/base/scalarFieldSmoother/`